spec/validations/manifest_spec.rb

```diff
@@ -128,17 +128,11 @@ def create_package(parameter_hash)
 
   it 'should not error when using {{setting.}}' do
     package = create_package('defaultLocale' => 'en')
-    allow(package.manifest.location_options.first).to receive(:url) { '{{setting.test}}' }
-
-    expect(package).not_to have_error(/the url (.*) cannot use a setting because it is a signed url/)
-  end
-
-  it 'should error when using {{setting.}} with a signed url' do
-    package = create_package('defaultLocale' => 'en')
-    allow(package.manifest.location_options.first).to receive(:url) { '{{setting.test}}' }
-    allow(package.manifest.location_options.first).to receive(:signed) { true }
+    translation_files = double('AppFile', relative_path: 'translations/en.json')
+    allow(@package).to receive_messages(translation_files: [translation_files])
```

Comment:
Question: is there a better way to bypass the missing translation file validation for this test?